scripts/download_data.sh

```diff
@@ -161,6 +161,78 @@ then
     done
     rm -rf "$(dirname "$folder")"
   done
+elif [ "$DATASET_NAME" = "cf" ]
```

Comment:
Let's use the full name -- `codeforces`

Reply:
Also, maybe move codeforce data preprocessing to another script and just call it from this?